### PR TITLE
Adds kill objective confirmations.

### PR DESCRIPTION
**Pull request by AnturK**
Something i seen mentioned a few times. Adds new objective type, kill with confirmation. Same as usual but you need to have a photo of the body for it to count. Also restores mobs lying on photos.

This turned out to be more snowflake than i expected so going to clean this up a bit later.

How far do you think corpse identification should go ? Right now it's just face visible and not mangled.

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Instead of a photo you should get an innate action button that lets you peel the fingerprints off of corpses. Having fingerprints on you would be grounds for arrest by sec but wouldn't immediately tell them who was killed and where like a picture would. They'd have to scan the prints then look up the identity in the database.

Tl;dr have traitors rip off the fingerprints of their kill instead of taking a picture

**Comment by BebeYoshi** [comment]
If you kill them, and take a photo of their corpse, and afterwards they get cloned or revived somehow, would it still count? Also is this objective just for traitors or changelings as well? Cheers

**Comment by AnturK** [comment]
I was considering something other than photos, maybe i'll just make the proof method variable to see what works best. I felt photo gives some minimal deniability when found.

They still need to be dead at the round end, not necessarily from when you took the photo though.
Right now it's just backend didn't actually add it to any antags.

**Comment by Robustin** [comment]
FYI this won't work if you were to get someone as a target, that person has to leave so they ask admins to find a replacement (rare, but it happens), and then another player takes control.

**Comment by Repukan** [comment]
Why not just make the traitor bring a jar with the targets brain on it, or bring the head back.

**Comment by Barhandar** [comment]
Yeah, this seems to be redtext bait, due to how many deaths leave target mangled or bodyless (or outright brainless). Your target is a miner who died to megafauna/loot without cloning? Whoopsiedaisy you now must find the head on lavaland with megafauna right near it. Or you're SOL if it was Hierophant or Memento Mori. 

Besides, what's the point in confirmation if "Syndicate" already can distinguish who is alive, dead and marooned at the end of the round?

**Comment by XDTM** [comment]
Perhaps the target could be restricted to your own department, so you should almost always have a chance of tailing them before they disappear or get disintegrated.

**Comment by bgobandit** [comment]
I like this idea a lot but weirdly I am more concerned with gameplay/meta, i.e. getting a camera. Photos have no non-frivolous uses currently and there are limited cameras lying around the station so I can easily see shit like security confiscating all the cameras or cargo refusing to autolathe one because "you could be a traitor" etc.

**Comment by Dorsisdwarf** [comment]
Cameras are *already* used for sneakily completing the document theft objective and sec dont care about them

**Comment by davethwave** [comment]
You can make cameras via a public lathe, the exclusion to this is box since the lathe there is in cargo.

**Comment by SouDescolado** [comment]
You get a core extraction thing when you get the objective to steal the core. Just make it so you spawn with the camera if you get one of those objectives.

**Comment by AnturK** [comment]
Will work on this some other time.
